ERCS/erc-77xx.md

```diff
@@ -0,0 +1,255 @@
+---
+eip: 77xx
```

Comment:
```suggestion
eip: 7720
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.